Dockerfile

```diff
@@ -6,6 +6,8 @@ FROM php:8.3.6-fpm-alpine3.19
 
 COPY --from=composer /usr/bin/composer /usr/bin/composer
 
+ENV COMPOSER_ALLOW_SUPERUSER=1
+
```

Comment:
No prefere set a non-root user with `USER {{ custom user }}` before installing and swap back to root with `USER root`